src/plugins/data/public/ui/filter_bar/filter_editor/value_input_type.tsx

```diff
@@ -147,6 +147,17 @@ class ValueInputTypeUI extends Component<Props> {
     this.props.onChange(boolValue);
   };
 
+  private onNumberChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const params = event.target.value;
+    let numValue;
+    if (typeof params === 'string') {
+      numValue = parseFloat(params);
+    } else if (typeof params === 'bigint') {
+      numValue = (params as BigInt).toString();
```

Comment:
We shouldn't have this. If the value is a BigInt, we should leave it as it.
```suggestion

```

Reply:
I added this logics according to previous logics on line 71, https://github.com/opensearch-project/OpenSearch-Dashboards/pull/7022/files/6b7d63232ed9b7b0c8a152ad66c262d41d8b4629#diff-d1ea4f28ecf59eb6ec2702dd4f29cd9a42aeab29c6065fc80c73be1f3ad9825aR71

I see they called `toString()` if the value is Bigint